### DISCUSSION TITLE
Verb chunk?

### DISCUSSION THREAD

**Discussion by HCSS-StratBase** [explosion/spaCy#12300] [Help: Other Questions] [answered]
Is there really no equivalent for verbs of what noun chunks is for nouns in spaCy itself (as opposed to textacy)? Or does somebody at least have some PoS/dependency combo patterns that could be used to obtain decent verb phrases?

**Comment by svlandeg** [comment, marked as answer]
Hi! There is no built-in functionality for retrieving verb phrases, but you can implement your own. In case it's helpful as background information, there's a [video](https://www.youtube.com/watch?v=BoyLPiXXEYA) by Vincent explaining some of the core concepts you'll need to pull this together. If you find something satisfying, we'd appreciate you posting back to the forum to help others, thanks!
